Source/WebCore/css/CSSCounterStyleRegistry.cpp

```diff
@@ -164,4 +164,65 @@ bool CSSCounterStyleRegistry::operator==(const CSSCounterStyleRegistry& other) c
     return m_authorCounterStyles == other.m_authorCounterStyles;
 }
 
+bool isCounterStyleUnsupportedByUserAgent(CSSValueID valueID)
+{
+    // This list has to be in sync with styles defined in counterStyles.css, once a style is added there, remove it from here.
+    switch (valueID) {
+    // FIXME: transform hard-coded styles into @counter-styles when possible (rdar://106708729).
+    case CSSValueBinary:
+    case CSSValueLowerHexadecimal:
+    case CSSValueOctal:
+    case CSSValueOriya:
+    case CSSValueUrdu:
+    case CSSValueUpperHexadecimal:
+    case CSSValueAfar:
+    case CSSValueEthiopicHalehameAaEt:
+    case CSSValueEthiopicHalehameAaEr:
+    case CSSValueAmharic:
+    case CSSValueEthiopicHalehameAmEt:
+    case CSSValueAmharicAbegede:
+    case CSSValueEthiopicAbegedeAmEt:
+    case CSSValueCjkEarthlyBranch:
+    case CSSValueCjkHeavenlyStem:
+    case CSSValueEthiopic:
+    case CSSValueEthiopicHalehameGez:
+    case CSSValueEthiopicAbegede:
+    case CSSValueEthiopicAbegedeGez:
+    case CSSValueHangulConsonant:
+    case CSSValueHangul:
+    case CSSValueLowerNorwegian:
+    case CSSValueOromo:
+    case CSSValueEthiopicHalehameOmEt:
+    case CSSValueSidama:
+    case CSSValueEthiopicHalehameSidEt:
+    case CSSValueSomali:
+    case CSSValueEthiopicHalehameSoEt:
+    case CSSValueTigre:
+    case CSSValueEthiopicHalehameTig:
+    case CSSValueTigrinyaEr:
+    case CSSValueEthiopicHalehameTiEr:
+    case CSSValueTigrinyaErAbegede:
+    case CSSValueEthiopicAbegedeTiEr:
+    case CSSValueTigrinyaEt:
+    case CSSValueEthiopicHalehameTiEt:
+    case CSSValueTigrinyaEtAbegede:
+    case CSSValueEthiopicAbegedeTiEt:
+    case CSSValueUpperGreek:
+    case CSSValueUpperNorwegian:
+    case CSSValueAsterisks:
+    case CSSValueFootnotes:
+    case CSSValueCjkIdeographic:
+    case CSSValueSimpChineseInformal:
+    case CSSValueSimpChineseFormal:
+    case CSSValueTradChineseInformal:
+    case CSSValueTradChineseFormal:
+    case CSSValueEthiopicNumeric:
+    case CSSValueCustomCounterStyle:
```

Comment:
does that value really exist?

Reply:
yes, in your renaming patch you haven't changed the value in CSSValueKeywords.in, but actually we don't need this value, but just the ListStyle::Type counter part. I'll remove it from the keywords!